src/Networks/Blockcore.Networks.RoyalSportsCity/RoyalSportsCitySetup.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+using System.Collections.Generic;
+using Blockcore.Consensus.Checkpoints;
+using Blockcore.Networks.RoyalSportsCity.Networks;
+using Blockcore.Networks.RoyalSportsCity.Networks.Setup;
+using NBitcoin;
+
+namespace Blockcore.Networks.RoyalSportsCity
+{
+    internal class RoyalSportsCitySetup
+    {
+        internal static RoyalSportsCitySetup Instance = new RoyalSportsCitySetup();
+
+        internal CoinSetup Setup = new CoinSetup
+        {
+            FileNamePrefix = "royalsportscity",
+            ConfigFileName = "royalsportscity.conf",
+            Magic = "01-4D-59-43",
```

Comment:
Do not reuse Magic values from different networks. Cybits is already using these magic bytes. The magic bytes is responsible for making sure that node of "chain x" does not connect to "chain y".

Reply:
I changed it to "01-52-53-43"